modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -99,6 +99,29 @@ using GRunArg  = util::variant<
     >;
 using GRunArgs = std::vector<GRunArg>;
 
+// TODO: Think about the addition operator
+/**
+ * @brief This operator allows to complement input vector at runtime.
+ *
+ * It's an ordinary overload of addition assignment operator.
+ *
+ * Example of usage:
+ *
+ * @code
+ * cv::Mat inMat;
+ * GRunArgs inVec;
+ * inVec += cv::gin(inMat);
```

Comment:
Code snippets should go via `@snippet`, the goal is to have that code compile and so valid.

Reply:
Unfortunately it's impossible to use `@snippet` because `EXAMPLE_PATH`  is set as `${CMAKE_SOURCE_DIR}/samples`

So I just deleted it

Reply:
`EXAMPLE_PATH` for master branch (w/o opencv_contrib) is below and contains `modules/gapi/samples` entry:

```
EXAMPLE_PATH           = /home/alalek/projects/opencv/dev/samples  \
 /home/alalek/projects/opencv/dev/modules/core/doc \
/home/alalek/projects/opencv/dev/modules/imgproc/doc \
/home/alalek/projects/opencv/dev/modules/videoio/doc \
/home/alalek/projects/opencv/dev/modules/highgui/doc \
/home/alalek/projects/opencv/dev/modules/video/doc \
/home/alalek/projects/opencv/dev/modules/calib3d/doc \
/home/alalek/projects/opencv/dev/modules/features2d/doc \
/home/alalek/projects/opencv/dev/modules/objdetect/doc \
/home/alalek/projects/opencv/dev/modules/ml/doc \
/home/alalek/projects/opencv/dev/modules/stitching/doc \
/home/alalek/projects/opencv/dev/modules/gapi/doc  \
 /home/alalek/projects/opencv/dev/modules/gapi/samples
```